README.md

```diff
@@ -31,11 +31,16 @@ You may also visit our [Quantum](https://github.com/microsoft/quantum) repositor
 
 ## Building from Source ##
 
-Before you can build the source code on this repository and start contributing to the Q# compiler and extensions you need to run the PowerShell script [bootstrap.ps1](./bootstrap.ps1) to set up your environment.
+Before you can build the source code on this repository and start contributing to the Q# compiler and extensions you need to run the PowerShell script [bootstrap.ps1](./bootstrap.ps1) and a few more commands to set up your environment.
 We refer to the [PowerShell GitHub repository](https://github.com/powershell/powershell) for instructions on how to install PowerShell.
 The script in particular generates the files that are needed for building based on the templates in this repository.
+```
+pwsh ./bootstrap.ps1
+git submodule init
+git submodule update

```

Comment:
There was some discussion earlier with @samarsha and @ScottCarda-MS as to whether we should just add these lines to the bootstrap.ps1 itself. I know I'm personally in favor of that such that bootstrap remains the one stop solution for preparing the repo. I think that would be worth doing now, since we know anyone who newly clones the repo would run into something similar.

Reply:
Changed.